markdown/src/main/java/com/nflabs/zeppelin/markdown/Markdown.java

```diff
@@ -50,6 +50,8 @@ public InterpreterResult interpret(String st) {
       html = md.process(st);
     } catch (IOException e) {
       return new InterpreterResult(Code.ERROR, e.getMessage());
+    } catch (java.lang.RuntimeException e) {
```

Comment:
Since this throw the exact error as `catch (IOException e)` why not consider to to do something like:
```
 } catch (IOException | RuntimeException  e) {
    return new InterpreterResult(Code.ERROR, e.getMessage());
}
```

instead?